core/src/main/scala/io/finch/Outputs.scala

```diff
@@ -4,47 +4,66 @@ import com.twitter.finagle.http.Status
 
 trait Outputs {
 
-  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list is assembled.
+  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list
+  // is assembled.
 
   // 2xx
-  def Ok[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Ok)                                               // 200
-  def Created[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Created)                                     // 201
-  def Accepted[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Accepted)                                   // 202
-  def NoContent[A](a: A): Output.Payload[A] = Output.Payload(a, Status.NoContent)                                 // 204
+  def Ok[A](a: A): Output[A] = Output.payload(a, Status.Ok)                                   // 200
+  def Created[A](a: A): Output[A] = Output.payload(a, Status.Created)                         // 201
+  val Accepted: Output[Unit] = Output.unit(Status.Accepted)                                   // 202
```

Comment:
Providing info in headers definitely a valid solution and may well be the right answer given how easy it is to write `Output.payload` if you need to provide more. I think in the example you used, I would tend toward a more CQRS approach and have an `Endpoint[UserCreationStarted]` or something like that. But I guess its not really our place to push this kind of approach on Finch users, so keeping it as `Unit` is fine.